./getDistances.py >> results/res 2>> results/err